metadata/src/main/java/edu/unc/lib/dl/reporting/ActivityMetricsClient.java

```diff
@@ -86,11 +86,23 @@ public void incrFinishedDeposit() {
 		}
 	}
 	
-	public void incrDepositFileThroughput(long bytes) {
+	public void incrDepositFileThroughput(String uuid, long bytes) {
 		try (Jedis jedis = getJedisPool().getResource()) {
 			String date = metricsDateFormat.format(new Date());
 			jedis.hincrBy(DEPOSIT_METRICS_PREFIX + date, "throughput-files", 1);
```

Comment:
How do you feel about continuing to track the cumulative throughput for the day when we are also recording per deposit/file statistics?  With the new setup its a nice side effect that ingests that run overnight will end up with metrics on the correct day rather than the whole deposit going onto whatever day it finishes on.  So if it is just as useful for your visualization purposes, I don't see a reason that we would need to keep the cumulative as well.